okhttp/src/main/kotlin/okhttp3/ConnectionPool.kt

```diff
@@ -39,13 +43,42 @@ class ConnectionPool internal constructor(
     timeUnit: TimeUnit = TimeUnit.MINUTES,
     taskRunner: TaskRunner = TaskRunner.INSTANCE,
     connectionListener: ConnectionListener = ConnectionListener.NONE,
+    readTimeoutMillis: Int = 10_000,
```

Comment:
I wonder about these defaults, and that they’re duplicated here and in OkHttpClient.Builder.

Not a dealbreaker, just a thing I’m contemplating